torchbenchmark/models/lennard_jones/__init__.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, test, device, jit=False, batch_size=None, extra_args=[]):
         )
         self.model = self.model.to(device)
 
-        r = torch.linspace(0.5, 2 * sigma, steps=self.batch_size, requires_grad=True)
+        r = torch.linspace(0.5, 2 * sigma, steps=self.batch_size)
```

Comment:
This is a bugfix -- the inputs to the model should not require grad, the weights of the model do.